app/main/views/g_cloud.py

```diff
@@ -500,6 +501,7 @@ def view_project(framework_family, project_id):
 
         search = None
         buyer_search_page_url = None
+        search_summary_count = 0
```

Comment:
Default None might be better in case someone was tempted to do some math with that zero :-)

Can we call it `search_results_count` as we're not counting summaries..

Reply:
Done